config/galaxy.ini.sample

```diff
@@ -203,7 +203,8 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # Pass debug flag to conda commands.
 #conda_debug = False
 # conda channels to enable by default (http://conda.pydata.org/docs/custom-channels.html)
-#conda_ensure_channels = conda-forge,r,bioconda,iuc
+# the recommended channel order is the one from BioConda (https://github.com/bioconda/bioconda-recipes/blob/master/config.yml#L8)
+#conda_ensure_channels = bioconda,r,defaults,conda-forge,iuc
```

Comment:
This really depends on what we want to use this channel for. My Intension was to put in old recipes, that are outdated and we don't want to have in bioconda, or some ugly hacks (but for new recipes). Not replacing/overwriting recipes from other channels.

Up for discussion I suppose.